consensus/common/src/main/java/org/hyperledger/besu/consensus/common/ForksSchedule.java

```diff
@@ -42,13 +41,14 @@
   }
 
   @VisibleForTesting
-  public BftForksSchedule(final ForkSpec<C> genesisFork, final Collection<ForkSpec<C>> forks) {
+  public ForksSchedule(final ForkSpec<C> genesisFork, final Collection<ForkSpec<C>> forks) {
     this.forks.add(genesisFork);
     this.forks.addAll(forks);
   }
 
-  public static <T extends BftConfigOptions, U extends BftFork> BftForksSchedule<T> create(
-      final T initial, final List<U> forks, final BftSpecCreator<T, U> specCreator) {
+  public static <T extends BftConfigOptions, U extends BftFork>
+      ForksSchedule<T> createForBftConfigOptions(
```

Comment:
I'm thinking this should either be made generic or be moved out to another class?

Reply:
This really quite specific to bft transitions. So think it should be out to a Bft specific class

Reply:
done!